gpkit/constraints/relax.py

```diff
@@ -105,8 +105,11 @@ def __init__(self, constraints, include_only=None, exclude=None):
         exclude = frozenset(exclude) if exclude else frozenset()
         include_only = frozenset(include_only) if include_only else frozenset()
         substitutions = KeyDict(constraints.substitutions)
-        constants, _, _ = parse_subs(constraints.varkeys,
-                                     constraints.substitutions)
+        constants, _, linked = parse_subs(constraints.varkeys, substitutions)
+        if linked:
+            kdc = KeyDict(constants)
+            constants.update({v: f(kdc) for v, f in linked.items()})
+        self.constants = constants
```

Comment:
so was the ConstantsRelaxed not recognizing linked substitutions and that's why it was having problems creating?

Reply:
Yup!

Reply:
so why didn't it have any problems after an attempted solve

Reply:
I think that the last value of the linked parameters might have been then present? Not the intended behaviour, now that you point it out...

Reply:
uh interesting.

Reply:
created issue #1207 for that